Makefile

```diff
@@ -0,0 +1,34 @@
+# HELP
+# This will output the help for each task
+# thanks to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+.PHONY: help
+
+help: ## This help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.DEFAULT_GOAL := help
+
+run-local:
+	@echo '(+) Starting to build LOCAL env'
+	go run main.go
+
+setup-local:
+	@echo '(+) Setting up go modules in env'
+	go get -u github.com/gorilla/mux
+	go get -u github.com/lib/pq
+	go get -u github.com/stretchr/testify/assert
+	go get -u github.com/go-sql-driver/mysql
```

Comment:
Did you encounter any problem creating a `go.mod` file and adding an instruction here to download the dependencies?

Reply:
Yeah for some reason the github.com/go-sql-driver/mysql was having issues running telling:
`cannot find module providing package github.com/go-sql-driver/mysql`